### PR TITLE
NMS-12809: updating user documentation

### PR DESCRIPTION
**Pull request by Bonrob2**
### All Contributors

* [ ] Have you read and followed our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [ ] Have you [made an issue in the OpenNMS issue tracker](https://issues.opennms.org)?<br>If so, you should:
  1. update the title of this PR to be of the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
* [ ] Have you made a comment in that issue which points back to this PR?
* [ ] Have you updated the JIRA link at the bottom of this comment to link to your issue?
* [ ] If this is a new or updated feature, is there documentation for the new behavior?
* [ ] If this is new code, are there unit and/or integration tests?
* [ ] If this PR targets a `foundation-*` branch, does it avoid changing files in `$OPENNMS_HOME/etc/`?

### Pull Request Process

One or more reviewers should be assigned to each PR.

If you know that a particular person is subject matter expert in the area your PR affects, feel free to assign one or more reviewers when you create this PR, otherwise reviewers will be assigned for you.

If you have made additions or changes to the documentation, or if you _need_ documentation for these code changes, please make sure a technical writer has looked it over.

Once the reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the Order of the Green Polo) you are welcome to merge the PR.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-12809

### CODE REVIEW

**Comment by Bonrob2** [comment]
@ronny-trommer Draft request. There are still a few questions highlighted in yellow. 
Going forward I will plan to PR on smaller chunks of text. That should allow for less material to go through and a quicker turnaround. Dragons!

**Comment by RangerRick** [comment]
Should this be `foundation-2019`?

**Comment by Bonrob2** [comment]
> Should this be `foundation-2019`?

No. This is restructuring of the documentation for release 27.

**Comment by RangerRick** [comment]
ok, cool, just checking 👍

File: opennms-doc/guide-admin/src/asciidoc/text/user-management/user-groups.adoc [hunk 1]
```diff
@@ -0,0 +1,25 @@
+
+// Allow GitHub image rendering
+:imagesdir: ../../images
+
+[[ga-user-groups]]
+=== Groups
+
+A group is a collection of users. 
+Organziging users into groups helps with notifications and allows you to assign a set of users to on-call roles to build more complex notification workflows.
```
Comment (j-white) [inline opennms-doc/guide-admin/src/asciidoc/text/user-management/user-groups.adoc:9]:
```suggestion
Organizing users into groups helps with notifications and allows you to assign a set of users to on-call roles to build more complex notification workflows.
```

File: opennms-doc/guide-admin/src/asciidoc/text/user-management/user-groups.adoc [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+[[ga-user-group-create]]
+==== Creating a User Group
+
+ . Log in as a user with administrative permissions.
+. Click the gear icon in the top right. 
+. Choose *Configure OpenNMS>Configure Users, Groups and On-Call roles* and select *Configure Groups*.
+. Specify a group name and description and click *OK*.
+. Add users to the group by selecting them from the *Available Users* column and using the arrows to move them to the *Currently in Group* column.
+. _(Optional)_ Assign categories of responsibility to the group, such as Routers, Switchers, Servers, etc. 
+. _(Optional)_ Create a ink:ga-user-schedule[duty schedule].
```
Comment (j-white) [inline opennms-doc/guide-admin/src/asciidoc/text/user-management/user-groups.adoc:20]:
```suggestion
. _(Optional)_ Create a link:ga-user-schedule[duty schedule].
```

File: opennms-doc/guide-admin/src/asciidoc/text/user-management/user-groups.adoc [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+[[ga-user-group-create]]
+==== Creating a User Group
+
+ . Log in as a user with administrative permissions.
+. Click the gear icon in the top right. 
+. Choose *Configure OpenNMS>Configure Users, Groups and On-Call roles* and select *Configure Groups*.
+. Specify a group name and description and click *OK*.
+. Add users to the group by selecting them from the *Available Users* column and using the arrows to move them to the *Currently in Group* column.
+. _(Optional)_ Assign categories of responsibility to the group, such as Routers, Switchers, Servers, etc.
```
Comment (j-white) [inline opennms-doc/guide-admin/src/asciidoc/text/user-management/user-groups.adoc:19]:
```suggestion
. _(Optional)_ Assign categories of responsibility to the group, such as Routers, Switches, Servers, etc. 
```

File: opennms-doc/guide-admin/src/asciidoc/text/user-management/user-config.adoc [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+WARNING: Do not delete the default _admin_ and _rtc_ users.
+         The _rtc_ user is used for the communication of the Real-Time Console on the start page to calculate the node and service availability.
+
+[[ga-user-create]]
+==== Creating a User
+
+. Log in as a user with administrative permissions.
+. Click the gear icon in the top right. 
+. Choose *Configure OpenNMS>Configure Users, Groups and On-Call roles* and select *Configure Users*.
```
Comment (indigo423) [inline opennms-doc/guide-admin/src/asciidoc/text/user-management/user-config.adoc:25]: just consistency thing, I guess we did in other places something like this here:

```
"Configure OpenNMS -> Configure Users, Groups and On-Call roles"
```
This is the spot I've looked it up: https://github.com/OpenNMS/opennms/edit/develop/opennms-doc/guide-install/src/asciidoc/text/basic-setup-opennms/rhel/step-4-change-admin-password.adoc

File: opennms-doc/guide-admin/src/asciidoc/text/user-management/user-config.adoc [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+* group on duty at time of notification – all users also on duty receive notification
+* group on duty, no member users on duty – notification is queued and sent to the next user who comes on duty
+* off-duty group – notification never sent
+
+To add a duty schedule for a user (or group), follow these steps:
+
+. Log in as a user with administrative permissions.
+. Click the gear icon in the top-right. 
+. Choose *Configure OpenNMS>Configure Users, Groups and On-Call roles* and select *Configure Users* (*Configure Groups*).
```
Comment (indigo423) [inline opennms-doc/guide-admin/src/asciidoc/text/user-management/user-config.adoc:58]: same here

File: opennms-doc/guide-admin/src/asciidoc/text/user-management/user-groups.adoc [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+. Choose *Configure OpenNMS>Configure Users, Groups and On-Call roles* and select *Configure Groups*.
+. Specify a group name and description and click *OK*.
+. Add users to the group by selecting them from the *Available Users* column and using the arrows to move them to the *Currently in Group* column.
+. _(Optional)_ Assign categories of responsibility to the group, such as Routers, Switchers, Servers, etc. 
+. _(Optional)_ Create a ink:ga-user-schedule[duty schedule].
+. Click *Finish*. 
+
+NOTE: Users will receive notifications in the order in which they appear in the group. 
+
+#What happens when you delete a group? Does anyone receive notification that the group has been deleted? What if there are duty schedules. No notifications would end up going out. Is there an alert saying the group is associated with a notification or schedule?#
```
Comment (indigo423) [inline opennms-doc/guide-admin/src/asciidoc/text/user-management/user-groups.adoc:25]: #TODO here right?

File: opennms-doc/guide-admin/src/asciidoc/text/user-management/user-maintenance.adoc [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+
+.*Deleting a user or group*
+
+. Log in as a *User* with administrative permissions
+. Click the gear icon in the top right. 
+. Choose *Configure OpenNMS>Configure Users, Groups and On-Call roles* and select *Configure Users* (*Configure Groups*).
+. Click the trash bin icon beside the user (or group) you want to delete. 
+. Confirm delete request with *OK*.
+
+#are there issues with permissions and deleting users or groups?#
```
Comment (indigo423) [inline opennms-doc/guide-admin/src/asciidoc/text/user-management/user-maintenance.adoc:37]: #TODO here right?

File: opennms-doc/guide-admin/src/asciidoc/text/user-management/users.adoc [hunk 8]
```diff
@@ -2,69 +2,63 @@
...
+* Number of alarms in the `alarms` table
+* Number of events in the `events` table
+* Number of IP interfaces in the `ipinterface` table
+* Number of nodes in the `node` table
+* Number of nodes, grouped by System OID
+
+[[ga-admin-user-setup]]
+==== Admin User Setup
+
+After logging in for the first time, make sure to change the default admin user password to a secure one:
```
Comment (indigo423) [inline opennms-doc/guide-admin/src/asciidoc/text/user-management/users.adoc:52]: Ah here it is the admin password part :)

File: opennms-doc/guide-admin/src/asciidoc/text/about/about.adoc [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+|`code` | Anything you must type or enter, and the names for code-related elements (classes, methods, commands).
+
+
+|===
+
+[[ga-admin-help]]
+=== Need Help?
+
+* contact 
+* join our community on xref:https://opennms.discourse.group/latest[Discourse]
```
[the remark this answers is not in the document]
Comment (Bonrob2) [reply opennms-doc/guide-admin/src/asciidoc/text/about/about.adoc:52]: Updated.

File: opennms-doc/guide-admin/src/asciidoc/text/user-management/security-roles.adoc [hunk 10]
```diff
@@ -3,59 +3,64 @@
...
-| _ROLE_ANONYMOUS_    | Allows _HTTP OPTIONS_ request to show allowed HTTP methods on a ReST resources and the login and logout page of the Web User Interface.
-| _ROLE_ADMIN_        | Permissions to create, read, update and delete in the Web User Interface and the _ReST API_.
-| _ROLE_ASSET_EDITOR_ | Permissions to just update the asset records from nodes.
-| _ROLE_DASHBOARD_    | Allow users to just have access to the _Dashboard_.
-| _ROLE_DELEGATE_     | Allows actions (such as acknowledging an alarm) to be performed on behalf of another user.
-| _ROLE_JMX_          | Allows retrieving JMX metrics but does not allow executing MBeans of the _{opennms-product-name}_ JVM, even if they just return simple values.
+| anyone            | If the `opennms-webapp-remoting` package is installed, any user can download the Java Webstart installation package for the remote poller from http://opennms.server:8980/opennms-remoting/webstart/app.jnlp.
+| *ROLE_ADMIN        | Permissions to create, read, update and delete in the web UI and the _ReST API_.
```
[the remark this answers is not in the document]
Comment (Bonrob2) [reply opennms-doc/guide-admin/src/asciidoc/text/user-management/security-roles.adoc:21]: Not just German. Updated. :-)

File: opennms-doc/guide-admin/src/asciidoc/text/user-management/user-config.adoc [hunk 11]
```diff
@@ -0,0 +1,62 @@
+
+// Allow GitHub image rendering
+:imagesdir: ../../images
+
+[[ga-user-config]]
+=== User Creation and Configuration
+
+Only a user with admin privileges can create users and assign security roles to them. 
+We recommend creating a new admin user instead of using the default.
```
[the remark this answers is not in the document]
Comment (Bonrob2) [reply opennms-doc/guide-admin/src/asciidoc/text/user-management/user-config.adoc:9]: Added a link to the "Admin User Setup" topic in a previous chapter that describes how to create a new admin user and change the default password.

File: opennms-doc/guide-admin/src/asciidoc/text/user-management/user-config.adoc (hunk 4 above)
Comment (Bonrob2) [reply opennms-doc/guide-admin/src/asciidoc/text/user-management/user-config.adoc:25]: I was considering changing this to the > only as I updated the docs, but am fine with the ->

**Pull request merged**